hybrid-standalone/modules/ROOT/pages/managing-properties-hybrid.adoc

```diff
@@ -0,0 +1,117 @@
+
+[IMPORTANT]
+====
+**Restart Requirement**: Modifying properties after application deployment requires restarting the application and any associated server groups or clusters to apply changes.
+====
+
+
+== Add Application Properties by Using a Properties File
+
+
+Include property placeholder YAML files in your deployable application archive to provide default configuration values. For comprehensive property file configuration guidance, see xref:mule-runtime::configuring-properties.adoc[Configuring Properties].
+
+Mule runtime engine loads file-based properties during application initialization.
+
+== Manage Properties Through Runtime Manager
+
+Configure and manage application properties dynamically through Runtime Manager's centralized property management interface, enabling environment-specific configuration without application code changes.
+
+
+=== Add Application properties in Runtime Manager
+
+Navigate to the property management interface for your deployed application:
+
+. From Anypoint Platform, select *Runtime Manager*.
+. Click the *Applications* tab.
+. Click the *Status* column to display the details pane.
+. Click *Manage Application*.
+. On the *Settings* page, click the *Properties* tab.
+
+.The screenshot shows (*1*) the *Properties* tab, (*2*) the *Key* and *Value* fields, and (*3*) the *...* menu on the application *Settings* page.
+image::rtm-manage-props.png["*Properties* tab, *Key* and *Value* fields, and *...* menu on the *Settings* page"]
+
+=== Property Configuration Operations
+
+**Create New Properties**
+
+. Enter the property name in the *Key* field.
+. Enter the property value in the *Value* field.  
+. Click *Save* > *Redeploy Application*.
+
+**Modify Existing Properties**
+
+. Locate the property in the properties list.  
+. Click the *...* menu for property-specific actions.
+. Select *Edit* to modify the property value.
+. Click *Save* > *Redeploy Application*.
+
+**Delete Properties**
+
+. Locate the property in the properties list.
+. Click the *...* menu for the target property.
+. Select *Delete* to remove the property.
+. Click *Save* > *Redeploy Application*.
+
+[IMPORTANT]
+All property changes require application redeployment to take effect. Plan property modifications during appropriate maintenance windows to minimize service disruption.
+
+
+== Advanced Property Configuration Methods
+
+These are some alternative methods for overriding app properties:
+
+* Configure an external location to add property placeholders or secure property placeholder files to override properties.
+
+* Set Java system environment variables at deployment time to override properties.
++
+To do this, deploy the app using this command:
++
+----
+mule -M-Dsecret.key=toSecretPassword -M-Denv=prod -M-Ddb.password=secretPassword -app myApp.zip
+----
++
+In this case, all the values typed into the command are stored only in memory and must be provided every time. The values are never stored in any file.
```

Comment:
```suggestion
In this case, the system stores the input values only in memory. Because these values are never saved to a file, you must provide them for every command execution.
```